orion/core.go

```diff
@@ -513,6 +513,17 @@ func GetDefaultServer(name string, opts ...DefaultServerOption) Server {
 	return server
 }
 
+// GetDefaultServerWithConfigAndOpts returns a default server object that uses provided configuration
+func GetDefaultServerWithConfigAndOpts(config Config, opts ...DefaultServerOption) Server {
```

Comment:
There is a function present that accept options.
if you pass the server name, it should be able to load the config and use the passed opts.

GetDefaultServer(name string, opts ...DefaultServerOption) Server 

This doesn't addressed the issue?

Reply:
Config for ReadTimeout and WriteTimeout was not set. Updated.